src/stylesheets/components/_search-result.scss

```diff
@@ -2,3 +2,7 @@
   border-bottom: 1px solid $color-black;
   padding-bottom: 1em;
 }
+
+.usa-label {
```

Comment:
@joshbruce Made the requested changes. Let me know if there are further changes to be made. Thank you.